<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Parametrs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>568</width>
             <height>648</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <property name="horizontalSpacing">
               <number>5</number>
              </property>
              <property name="verticalSpacing">
               <number>3</number>
              </property>
              <item row="1" column="0">
               <widget class="QLineEdit" name="txtPathToPKL"/>
              </item>
              <item row="8" column="1">
               <widget class="QToolButton" name="toolButton_layer_refresh">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>refresh.png</normaloff>refresh.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QToolButton" name="toolButton_PKL">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_17">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Layer of destinations (points/polygons) </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="1">
               <widget class="QToolButton" name="toolButton_protocol">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="14" column="0">
               <widget class="QCheckBox" name="cbUseFields">
                <property name="text">
                 <string>Run aggregate</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_6"/>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Layer of origins (points/polygons)</string>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_18">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cmbLayersDest"/>
                </item>
                <item>
                 <widget class="QCheckBox" name="cbSelectedOnly2">
                  <property name="text">
                   <string>Selected features only</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QLineEdit" name="txtPathToProtocols"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Dataset folder</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Output folder</string>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QToolButton" name="toolButton_layer_dest_refresh">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>refresh.png</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
              <item row="17" column="0">
               <widget class="QComboBox" name="cmbFields"/>
              </item>
              <item row="8" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_19">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cmbLayers">
                  <property name="mouseTracking">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="cbSelectedOnly1">
                  <property name="text">
                   <string>Selected features only</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="15" column="0">
               <widget class="QLabel" name="lblFields">
                <property name="text">
                 <string>Field to aggregate</string>
                </property>
               </widget>
              </item>
              <item row="12" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Visualization layer</string>
                </property>
               </widget>
              </item>
              <item row="13" column="0">
               <widget class="QComboBox" name="cmbVizLayers"/>
              </item>
              <item row="13" column="1">
               <widget class="QToolButton" name="toolButton_viz_layers_refresh">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>refresh.png</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Min transfers                                           </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMinTransfers">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>10</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Max transfers                                          </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxTransfers">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>10</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="lbMaxWalkDistanceInitial">
                <property name="text">
                 <string>Max walk distance to the initial PT stop, m</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxWalkDist1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="leftMargin">
               <number>1</number>
              </property>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Max walk distance at transfer, m</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxWalkDist2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>10</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Max walk distance from a last PT stop, m</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxWalkDist3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>10</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_21">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Start at/Arrive before (time)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="dtStartTime">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="displayFormat">
                 <string>hh:mm:ss</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="lblDepartureInterval">
                <property name="text">
                 <string>Departure interval earliest, min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtDepartureInterval">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>20</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="lblMaxExtraTime">
                <property name="text">
                 <string>Maximum extra time at a first stop, min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxExtraTime">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>20</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Walking speed (km/h)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtSpeed">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>20</verstretch>
                 </sizepolicy>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhDialableCharactersOnly|Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="lblMaxWaitTime">
                <property name="text">
                 <string>Maximal waiting time at initial stop, min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxWaitTime">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>20</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Maximal waiting time at transfer stop, min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxWaitTimeTransfer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>20</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Maximal time travel, min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaxTimeTravel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>20</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="lblTimeInterval">
                <property name="text">
                 <string>Time interval between stored maps (min)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtTimeInterval">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>20</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textLog"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="textInfo">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBreakOn">
       <property name="text">
        <string>Break</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="gridFrame">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblMessages">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtPathToPKL</tabstop>
  <tabstop>toolButton_PKL</tabstop>
  <tabstop>txtPathToProtocols</tabstop>
  <tabstop>toolButton_protocol</tabstop>
  <tabstop>textInfo</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnBreakOn</tabstop>
  <tabstop>textLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
